app/src/main/java/com/door43/translationstudio/newui/translate/FirstTabFragment.java

```diff
@@ -118,6 +119,7 @@ public void onConfirmTabsDialog(String targetTranslationId, String[] sourceTrans
                 if(targetTranslation != null) {
                     try {
                         targetTranslation.addSourceTranslation(sourceTranslation);
+                        TargetTranslationMigrator.mergeInvalidChunksFromProject(AppContext.getLibrary(), targetTranslation);
```

Comment:
I'm not sure we need to merge the chunks here. What was your reasoning behind this?